imports/components/controls.jsx

```diff
@@ -0,0 +1,204 @@
+import { DropdownButton, MenuItem, Modal, Button } from 'react-bootstrap';
+import { PlayerList } from './player_list';
+
+class RevealControl extends React.Component {
```

Comment:
Hmm. I'd recommend adding `propTypes` declarations? They make for good documentation